en/common/command-line.mdown

```diff
@@ -17,41 +17,55 @@ This installs a tool named "parse" to `/usr/local/bin/parse`.
-The downloaded executable is named like: `parse-windows-x.xx.xx.exe`. 
-You can rename it to `parse.exe` and move it to an accessible location, so that you can run it by typing `parse` at your command prompt.
-
-### Updating
+The Parse command line tool for Windows is available [here](https://github.com/ParsePlatform/parse-cli/releases/latest).
+Download the windows executable named: `parse.exe` at this link. Note that this is not an installer, it is just a plain windows executable.
 
-You can update the command line tool using `parse update`. It will automatically update your Parse command line tool to the latest version.
+## Creating a Parse app
 
-For instance, if you run this command on Mac OS:
+You can create a new Parse app using `parse new`.
+It asks you a series of questions and at the end of it, you will have a new Parse app (with a given name).
+Additionally, you will also set up a cloud code project for the app.
+You can test that everything works by doing a `parse deploy` and executing the curl command printed at the end of output.
 
 ```bash
-$ parse update
-Downloading binary from https://parse.com/downloads/cloud_code/cli/parse-osx/latest.
-Successfully updated binary at: /usr/local/bin/parse
-```
+$ parse new
+Would you like to create a new app, or add Cloud Code to an existing app?
+Type "(n)ew" or "(e)xisting": n
+Please choose a name for your Parse app.
+Note that this name will appear on the Parse website,
+but it does not have to be the same as your mobile app's public name.
+Name: MyParseApp
+Awesome! Now it's time to setup some Cloud Code for the app: "MyParseApp",
+Next we will create a directory to hold your Cloud Code.
+Please enter the name to use for this directory,
+or hit ENTER to use "MyParseApp" as the directory name.
+
+Directory Name: my_parse_app
+Successfully configured email for current project to: "${ACCOUNT_EMAIL}"
+Your Cloud Code has been created at ${CUR_DIR}/my_parse_app.
+Next, you might want to deploy this code with "parse deploy".
+This includes a "Hello world" cloud function, so once you deploy
+you can test that it works, with:
 
-Note that Parse command line tool is configured to update automatically to the latest stable version.
-So, manually running `parse update` is not needed anymore. Your Parse command line tool is always up-to-date.
+curl -X POST \
+ -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+ -H "X-Parse-REST-API-Key: ${REST_API_KEY}" \
+ -H "Content-Type: application/json" \
+ -d '{}' \
+ https://api.parse.com/1/functions/hello
+ ```
 
-### Setting Up Your App
+## Setting up Cloud Code
 
-The next step is to create a directory to store the code that you will run in the cloud. 
-The command `parse new` sets up this directory, and will prompt you to pick which app you are creating Cloud Code for:
+If you already created a Parse app (for example, through the website or apps api) and want to set up cloud code for the app
```

Comment:
Capitalize Cloud Code, uppercase all of API, and add a comma after "for the app".